test/expected/chunk_utils.out

```diff
@@ -140,15 +162,104 @@ WHERE h.schema_name = 'public' AND (h.table_name = 'drop_chunk_test1' OR h.table
  _timescaledb_internal | _hyper_3_18_chunk | table | default_perm_user
 (18 rows)
 
+-- next two calls of show_chunks should give same set of chunks as above when combined
+SELECT show_chunks('drop_chunk_test1');
+              show_chunks               
+----------------------------------------
+ _timescaledb_internal._hyper_1_1_chunk
+ _timescaledb_internal._hyper_1_2_chunk
+ _timescaledb_internal._hyper_1_3_chunk
+ _timescaledb_internal._hyper_1_4_chunk
+ _timescaledb_internal._hyper_1_5_chunk
+ _timescaledb_internal._hyper_1_6_chunk
+(6 rows)
+
+SELECT * FROM show_chunks('drop_chunk_test2');
+               show_chunks               
+-----------------------------------------
+ _timescaledb_internal._hyper_2_7_chunk
+ _timescaledb_internal._hyper_2_8_chunk
+ _timescaledb_internal._hyper_2_9_chunk
+ _timescaledb_internal._hyper_2_10_chunk
+ _timescaledb_internal._hyper_2_11_chunk
+ _timescaledb_internal._hyper_2_12_chunk
+(6 rows)
+
 CREATE VIEW dependent_view AS SELECT * FROM _timescaledb_internal._hyper_1_1_chunk;
 \set ON_ERROR_STOP 0
 SELECT drop_chunks(2);
 ERROR:  cannot drop table _timescaledb_internal._hyper_1_1_chunk because other objects depend on it
 SELECT drop_chunks(NULL::interval);
-ERROR:  can only use drop_chunks with an INTERVAL for TIMESTAMP, TIMESTAMPTZ, and DATE types
+ERROR:  older_than and newer_than timestamps provided to drop_chunks cannot both be NULL
```

Comment:
@erimatnor @svenklemm note that the error messages here differ after introducing `show_chunks`
I think the new ones make more sense in this context but just wanted to get your opinion on this as well.
Tests introduced at https://github.com/timescale/timescaledb/pull/861

Reply:
@Ngalstyan4 Sure, i didnt focus on the error messages themselves only that there are tests for NULL and that NULL doesnt crash the backend or produces weird errors. Feel free to improve the wording of the error messages.